ldk/javascript/src/user/index.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * The User aptitude gives access to Olive Helps user related information
+ */
+import { promisify } from "../promisify";
+
+export interface User {
+    /**
+     * Returns a JWT with the current username in the subject field
```

Comment:
You'll want to use the `@returns` annotation here so its parsed by typedoc I do believe.

Reply:
Thanks.  Will do.

Reply:
Done.